framer/LinearGradient.coffee

```diff
@@ -0,0 +1,76 @@
+{_} = require "./Underscore"
+{BaseClass} = require "./BaseClass"
+{Color} = require "./Color"
+
+class exports.LinearGradient extends BaseClass
+	constructor: (options = {}) ->
+		options.start ?= "black"
+		options.end ?= "white"
+		options.angle ?= 0
```

Comment:
I think it's clearer (and more consistent with the rest of the lib) to use `_.defaults` for this